contracts/iotube/MimoBasedCashier.sol

```diff
@@ -0,0 +1,70 @@
+pragma solidity ^0.8.20;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+interface IMimoFactory {
```

Comment:
the `getAmountsOut` is the method for mimo router. may should rename to IMimoRouter?